app/models/todo_item.rb

```diff
@@ -1,6 +1,17 @@
 class TodoItem < ApplicationRecord
   belongs_to :todo_list
+
+
+
+
+
+
+
+
+  
+  enum priority: [:high, :medium, :low]
   def completed?
     !completed_at.blank?
```

Comment:
use completed_at.present? instead of this negation

Reply:
Thank you will look into this.